api/model/Campaign.py

```diff
@@ -0,0 +1,17 @@
+from api.helper.db import db
+
+
+class CampaignModel(db.Model):
```

Comment:
We should probably also define a simple (for now) `user/account` model (`id`, `username` i.e. twitter username we get after auth and `campaigns`) that is mapped to the campaigns model since we can have the same campaign names for different users. So you might need to update the `CampaignModel` class to allow multiple campaigns with the same name, but tied to different users. We can also add a foreign key to the UserModel class and update the `CampaignModel` class to have a composite primary key made up of `name` and `user_id`

Reply:
The plan is to get the app dockerized, but for now, the campaign endpoint was exposed to ensure read/write to the database is fine. The user/account model will be considered once implementation begins.

Reply:
Alright. I'll create an issue for it so we can track it later.